sdk/healthdataaiservices/azure-health-deidentification/src/main/java/com/azure/health/deidentification/models/DeidentificationJob.java

```diff
@@ -110,113 +104,91 @@ public DeidentificationJob(SourceStorageLocation sourceLocation, TargetStorageLo
 
     /**
      * Get the name property: The name of a job.
-     * 
+     *
      * @return the name value.
      */
     @Generated
     public String getName() {
         return this.name;
     }
 
-    /**
-     * Get the sourceLocation property: Storage location to perform the operation on.
-     * 
-     * @return the sourceLocation value.
-     */
-    @Generated
-    public SourceStorageLocation getSourceLocation() {
-        return this.sourceLocation;
-    }
-
-    /**
-     * Get the targetLocation property: Target location to store output of operation.
-     * 
-     * @return the targetLocation value.
-     */
-    @Generated
-    public TargetStorageLocation getTargetLocation() {
-        return this.targetLocation;
-    }
-
     /**
      * Get the operation property: Operation to perform on the input documents.
-     * 
+     *
      * @return the operation value.
      */
     @Generated
-    public OperationType getOperation() {
+    public DeidentificationOperationType getOperation() {
```

Comment:
Let's name this `getOperationType()` as this is returning the operation type and not the operation.

Reply:
Will apply in followup